apps/web/.env

```diff
@@ -1,8 +1,5 @@
 SKIP_PREFLIGHT_CHECK=true
 REACT_APP_API_URL=
-REACT_APP_WIDGET_SDK_PATH=
 REACT_APP_ENVIRONMENT=dev
-GLOBAL_CONTEXT_PATH=
-WEB_CONTEXT_PATH=
```

Comment:
Not used yet, but we would need it in the future to allow different paths in the Docker solution and self hosted solution. Should I leave it even if it is not used yet?

Reply:
Let's remove it to avoid confusion